cli/block/cli.py

```diff
@@ -0,0 +1,59 @@
+"""
+Provide implementation of the command line interface's block commands.
+"""
+import sys
+
+import click
+from remme import Remme
+
+from cli.block.forms import GetBlockByIdentifierForm
+from cli.block.help import BLOCK_IDENTIFIER_ARGUMENT_HELP_MESSAGE
+from cli.block.service import Block
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_URL_ARGUMENT_HELP_MESSAGE,
+)
+from cli.utils import (
+    print_errors,
+    print_result,
+)
+
+
+@click.group('block', chain=True)
+def block_commands():
+    """
+    Provide commands for working with block.
+    """
+    pass
+
+
+@click.option('--id', type=str, required=True, help=BLOCK_IDENTIFIER_ARGUMENT_HELP_MESSAGE)
+@click.option('--node-url', type=str, help=NODE_URL_ARGUMENT_HELP_MESSAGE)
```

Comment:
Add `required=False` and the default value for the node URL.

Reply:
@anastasiia-bilova, done.